job-dsl-core/src/main/groovy/javaposse/jobdsl/dsl/helpers/wrapper/WrapperContext.groovy

```diff
@@ -264,13 +264,21 @@ class WrapperContext extends AbstractExtensibleContext {
     /**
      * Injects globally defined passwords as environment variables into the job.
      *
-     * @since 1.23
+     * @since 1.45
      */
     @RequiresPlugin(id = 'envinject')
-    void injectPasswords() {
```

Comment:
A breaking change is not an option. But we could mark the old method as deprecated, so it will be removed eventually.